python/dgl/sparse/sddmm.py

```diff
@@ -12,8 +12,8 @@ def sddmm(
     r"""Sampled-Dense-Dense Matrix Multiplication (SDDMM).
 
     ``sddmm`` multiplies two dense matrices :attr:`mat1` and :attr:`mat2`
-    at the nonzero locations of sparse matrix :attr:`A`. Values of :attr:`A`
-    is not considered during the computation.
+    at the nonzero locations of sparse matrix :attr:`A`. The result is
```

Comment:
not related to this PR. let's make it a habit to always fix small things in a quick fix PR.

Reply:
done